codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketCloseStatus.java

```diff
@@ -165,23 +165,24 @@
     //public static final WebSocketCloseStatus ABNORMAL_CLOSURE = valueOf(1006, "Abnormal closure");
     //public static final WebSocketCloseStatus TLS_HANDSHAKE_FAILED(1015, "TLS handshake failed");
 
-    private final int code;
-    private volatile String message;
+    private final int statusCode;
+    private volatile String reasonText;
 
-    private WebSocketCloseStatus(int code, String message) {
-        if (!isValidRfc6455Code(code)) {
-            throw new IllegalArgumentException("WebSocket close status code does NOT comply with RFC-6455: " + code);
+    private WebSocketCloseStatus(int statusCode, String reasonText) {
+        if (!isValidRfc6455Code(statusCode)) {
+            throw new IllegalArgumentException(
+                "WebSocket close status code does NOT comply with RFC-6455: " + statusCode);
         }
-        this.code = code;
-        this.message = message;
+        this.statusCode = statusCode;
+        this.reasonText = reasonText;
     }
 
-    public int getCode() {
-        return code;
+    public int code() {
```

Comment:
please revert this one as it would be a breaking change

Reply:
This is a new class, introduced in this PR. It could NOT cause breaking change.